### PR TITLE
Add documentation of the severity flag in Docker Scan plugin

### PR DESCRIPTION
**Pull request by glours**
Add the document for the recently added `--severity` flag in Docker Scan plugin

### CODE REVIEW

**Comment by usha-mandya** [comment]
Check with @glours before merging this PR.

File: engine/scan/index.md [hunk 1]
```diff
@@ -273,6 +274,47 @@ For more free scans that keep your images secure, sign up to Snyk at https://doc
 
 For more information about the vulnerability data, see [Docker Vulnerability Scanning CLI Cheat Sheet](https://goto.docker.com/rs/929-FJL-178/images/cheat-sheet-docker-desktop-vulnerability-scanning-CLI.pdf){: target="_blank" rel="noopener" class="_"}.
 
+### Limit the level of vulnerabilities displayed
```
Comment (usha-mandya) [inline engine/scan/index.md:277]: Add a blank line after the heading

File: engine/scan/index.md [hunk 2]
```diff
@@ -273,6 +274,47 @@ For more free scans that keep your images secure, sign up to Snyk at https://doc
 
 For more information about the vulnerability data, see [Docker Vulnerability Scanning CLI Cheat Sheet](https://goto.docker.com/rs/929-FJL-178/images/cheat-sheet-docker-desktop-vulnerability-scanning-CLI.pdf){: target="_blank" rel="noopener" class="_"}.
 
+### Limit the level of vulnerabilities displayed
+If you want to only display some level of vulnerabilities, the `--severity` flag allows you to choose between 3 levels of
```
Comment (usha-mandya) [inline engine/scan/index.md:278]: How about 
"Docker scan allows you to choose the level of vulnerabilities displayed in your scan report using the `--severity` flag. You can set the severity flag to `low`, `medium`, or` high` depending on the level of vulnerabilities you’d like to see in your report.  For example, if you set the severity level as `medium`, the scan report displays all vulnerabilities that are classified as medium and high.

File: engine/scan/index.md (hunk 1 above)
Comment (usha-mandya) [inline engine/scan/index.md:277]:
```suggestion
### Limiting the level of vulnerabilities displayed
```

Just to be consistent with the other headings

**Comment by usha-mandya** [comment]
@glours Is this ready to go live?

**Comment by glours** [comment]
@usha-mandya it should be alive with the stable release next Thursday

**Comment by glours** [comment]
And I'll update this PR because we added a new flag 😄

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**